### PR TITLE
Fix quality selector not disappearing upon selection in mobile devices

### PR DESCRIPTION
**Pull request by Dananji**

### CODE REVIEW

File: vendor/assets/javascripts/mediaelement/plugins/quality-avalon.js [hunk 1]
```diff
@@ -169,12 +169,14 @@
                     : '') +
                     '">') +
                   ((src.title || _quality) + '</label>') +
-                  '</li>';
+                  '</li>';
```
Comment (cjcolvar) [inline vendor/assets/javascripts/mediaelement/plugins/quality-avalon.js]: Could you remove this trailing whitespace please?
```suggestion
                  '</li>';
```

**Comment by Dananji** [comment]
Oh sorry, I didn't know I could commit the changes in here as well.

**Comment by cjcolvar** [comment]
No worries, either way works. 😄

**Comment by cjcolvar** [comment]
I'm going to go ahead and merge this.  The failing build is due to a previous commit and will be fixed by https://github.com/avalonmediasystem/avalon/pull/3631.

**Pull request merged**